docs/get-started/getting-started-gcp.md

```diff
@@ -0,0 +1,113 @@
+* [How to spin up a Dataproc Cluster Accelerated by GPU](http://localhost:4000/gcp-dataproc/#how-to-spin-up-a-dataproc-cluster-accelerated-by-gpu)
+* [Run a sample Pyspark or Scala ETL and XGBoost training Notebooks on a Dataproc Cluster Accelerated by GPU](http://localhost:4000/gcp-dataproc/#run-pyspark-and-scala-notebook-a-dataproc-cluster-accelerated-by-gpu)
+* [Submit the same sample ETL application as a Spark job to a Dataproc Cluster Accelerated by GPU](http://localhost:4000/gcp-dataproc/#submit-spark-jobs-to-a-dataproc-cluster-accelerated-by-gpu)
+
+
+ 
+## How to spin up a Dataproc Cluster Accelerated by GPU
+ 
+ You can use [Cloud Shell](https://cloud.google.com/shell) to execute shell commands that will create a Dataproc cluster. Cloud Shell contains command line tools for interacting with Google Cloud Platform, including gcloud and gsutil. Alternatively, you can install [GCloud SDK](https://cloud.google.com/sdk/install) on your laptop. From the Cloud Shell, users will need to enable services within your project. Enable the Compute and Dataproc APIs in order to access Dataproc, and enable the Storage API as you’ll need a Google Cloud Storage bucket to house your data. This may take several minutes.
+```bash
+gcloud services enable compute.googleapis.com
+gcloud services enable dataproc.googleapis.com
+gcloud services enable storage-api.googleapis.com
+``` 
+
+After command line environment is setup, log in to your GCP account. We can now create a Dataproc cluster with configuration mentioned below.
+The configuration will allow users to run any of the [notebooks demo](../demo/GCP) on GCP. Alternatively, users can also start a 2*2T4 worker nodes.
+* [GPU Driver](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/gpu) and [RAPIDS Acclerator for Apache Spark](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/rapids) through initialization actions
+* One 8-core master node and 5 32-core worker nodes
+* Four NVIDIA T4 to each worker nodes
+* [Local SSDs](https://cloud.google.com/dataproc/docs/concepts/compute/dataproc-local-ssds) is recommended to improve IO for Spark scratch places  
+* Component gateway enabled for accessing Web UIs hosted on the cluster
+* Configuration for [GPU scheduling and isolation](/get-started/yarn-gpu.html)
+
+
+```bash
+    export REGION=[Your Prefer GCP Region]
+    export GCS_BUCKET=[Your GCS Bucket]
+    export CLUSTER_NAME=[Your Cluster Name]
+    export NUM_GPUS=4
+    export NUM_WORKERS=5
+
+gcloud dataproc clusters create $CLUSTER_NAME  \
+    --region $REGION \
+    --image-version=preview-ubuntu \
+    --master-machine-type n1-standard-16 \
+    --num-workers $NUM_WORKERS \
+    --worker-accelerator type=nvidia-tesla-t4,count=$NUM_GPUS \
+    --worker-machine-type n1-highmem-32\
+    --num-worker-local-ssds 4 \
+    --initialization-actions gs://dataproc-initialization-actions/gpu/install_gpu_driver.sh,gs://dataproc-initialization-actions/rapids/rapids.sh \
```

Comment:
When exactly do we install the spark plugin? The rapids script in github for this, as linked above only deals with dask (cudf version 0.9.2 and cuda 10.0) which we don't support in any official release of the spark plugin. How is any of this working?

Reply:
the init action script hasn't been merge yet. Since it supports both Dask and Spark (GCP asked for that), I had spent some time to make sure Dask is working properly. The pull request is here. https://github.com/GoogleCloudDataproc/initialization-actions/pull/770

Reply:
update, this pull request has been merge.